packages/two_dimensional_scrollables/lib/src/table_view/table.dart

```diff
@@ -861,31 +861,41 @@ class RenderTableViewport extends RenderTwoDimensionalViewport {
         )!;
 
         Rect getColumnRect(bool consumePadding) {
+          final double leadingOffsetCorrection =
+              axisDirectionIsReversed(verticalAxisDirection)
+                  ? leadingCell.size.height
+                  : 0.0;
+          final double trailingOffsetCorrection =
+              axisDirectionIsReversed(verticalAxisDirection)
+                  ? trailingCell.size.height
+                  : 0.0;
```

Comment:
(totally optional) I wonder if this reads better using records and a single `?:`?

```dart
final ({double leading, double trailing}) offsetCorrection = axisDirectionIsReversed(verticalAxisDirection))
  ? (leading: leadingCell.size.height, trailing: trailingCell.size.height)
  : (leading: 0.0, trailing: 0.0);
```

Bellow you'd then use `offsetCorrection.leading` and `offsetCorrection.trailing`

Reply:
Nice! I do love using records! (wait until you see merged cells 🤣 )